### ISSUE TITLE
Why not still loading jsgrps.js?

### ISSUE THREAD

**Issue by jpdevries** [jpdevries/revolution#25]
https://github.com/jpdevries/revolution/blob/grunt-concat/manager/templates/default/browser/index.tpl#L22

I'm a little confused because it since now we are only loading modx.jsgrps-min.js (if `compress_js`) is off we load the regular files. So I think we can do away with `jsgrps.js` entirely but when I went to do that I saw this code and thought huh, that doesn't look right. But when I test it it seems ok...

Also see https://github.com/modxcms/revolution/pull/12611

**Comment by Mark-H** [comment]
Isn't this where it's being used? 

https://github.com/modxcms/revolution/pull/12611/files#diff-83c025a415e5a2858f946261f0679e48R503

**Comment by jpdevries** [comment]
I'm not sure what you mean by that? Looking at `index.tpl` it seems like it would still load `jsgrps.js` but that doesn't seem to happen and I don't understand why.

**Comment by jpdevries** [comment]
oh sorry...it took so long for the page to load it didn't jump to the line for a while yes that is where externals is but look at the template. Wouldn't that load regardless of `$externals`?

**Comment by Mark-H** [comment]
Yeah. That's browser/index.tpl though, not header.tpl, which probably explains why you don't see it happening. Should probably be removed either way as loading it should be handled via $externals not the tpl....

nice catch ;)

**Comment by jpdevries** [comment]
Ah ok that makes sense. Cool
